### PR TITLE
Promote feedback

### PR DESCRIPTION
**Pull request by felixsch**
* Add a issue template for the issue tracker
  * Add a feedback section to the README.md
  * Add information about feedback to the help output

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!--
+
+ PLEASE MAKE SURE NO SENSIBLE INFORMATION ABOUT YOUR
```
Comment (ikapelyukhin) [inline .github/ISSUE_TEMPLATE.md]: `SENSIBLE` -> `SENSITIVE` :smile_cat:

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ SYSTEM IS EXPOSED WHEN SUBMITTING FEEDBACK!
+
+ Thank you for considering giving feedback to the project!
```
Comment (kalabiyau) [inline .github/ISSUE_TEMPLATE.md]: 2 cents

Repository Mirroring Tool (RMT) is an open-source project and we can make it even better with your help and feedback.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+ When adding your feedback/bug please keep in mind:
+   - This is not a support channel.
```
Comment (kalabiyau) [inline .github/ISSUE_TEMPLATE.md]: `This is not a support channel.` -> `This is not a support channel. If you are having payed support from SUSE business - please go to ...`

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+     Please go to https://scc.suse.com/support for support.
+   - When posting bugs/feedback here, there is no guarantee that the issue
+     will be handled immediately. Depending on the topic and the availability
+     of the developers, the issue maybe resolved in undefined time or
+     even ignored.
```
Comment (kalabiyau) [inline .github/ISSUE_TEMPLATE.md]: I believe we are not going to ignore any issue, there is a chance we are going to react and close something we know is impossible at the moment. Let's, maybe, remove `or even ignored.` part ?

File: lib/rmt/cli/base.rb [hunk 5]
```diff
@@ -16,6 +16,10 @@ def help(shell, subcommand = false)
       class_options_help(shell)
 
       shell.say "Run '#{basename} help [COMMAND]' for more information on a command and its subcommands."
+
+      shell.say
+      shell.say 'Do you have suggestions for improvement? Let us know!'
```
Comment (kalabiyau) [inline lib/rmt/cli/base.rb]: Would love to get the opinion from @jezuhke and @thutterer - here it is quite important which tone/emotion message has. Is it inviting enough to provide feedback?

File: lib/rmt/cli/base.rb [hunk 6]
```diff
@@ -16,6 +16,10 @@ def help(shell, subcommand = false)
...
+      shell.say 'Check out https://github.com/SUSE/rmt#feedback'
```
Comment (kalabiyau) [inline lib/rmt/cli/base.rb]: @felixsch maybe we could link directly to a new issue? That would be -1 click

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (wstephenson) [reply .github/ISSUE_TEMPLATE.md]: Good catch.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (wstephenson) [reply .github/ISSUE_TEMPLATE.md]: s/payed/paid/

File: lib/rmt/cli/base.rb (hunk 6 above)
Comment (felixsch) [reply lib/rmt/cli/base.rb]: thought about this as well so yes why not

File: .github/ISSUE_TEMPLATE.md (hunk 4 above)
Comment (felixsch) [reply .github/ISSUE_TEMPLATE.md]: fair enough. just wanted to make clear there is also the chance it will not work out. But yes right might sound a little bit rude.

File: lib/rmt/cli/base.rb (hunk 5 above)
Comment (felixsch) [reply lib/rmt/cli/base.rb]: Or maybe something like: Let us know -> We would love to here from you

File: .github/ISSUE_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+ Repository Mirroring Tool (RMT) is an open-source project and we can make it
+ even better with your help and feedback.
+
+ When adding your feedback/bug please keep in mind:
+   - This is not a support channel. If you are having paid support from SUSE
+     business please go to https://scc.suse.com/support for support.
+   - When posting bugs/feedback here, there is no guarantee that the issue
+     will be handled immediately. Depending on the topic and the availability
+     of the developers, the issue maybe resolved in undefined time.
+   - Make sure that you don't expose any personal or private information.
```
Comment (thutterer) [inline .github/ISSUE_TEMPLATE.md]: I would maybe start this paragraph with sth like this:
`Keep in mind that your feedback and all communication around it will be publically accessable by anyone. So make sure ...`

Please rephrase it at will / with Will :gb:

File: .github/ISSUE_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<!--
+
+ PLEASE MAKE SURE NO SENSITIVE INFORMATION ABOUT YOUR
+ SYSTEM IS EXPOSED WHEN SUBMITTING FEEDBACK!
+
+ Repository Mirroring Tool (RMT) is an open-source project and we can make it
+ even better with your help and feedback.
+
+ When adding your feedback/bug please keep in mind:
+   - This is not a support channel. If you are having paid support from SUSE
```
Comment (lagartoflojo) [inline .github/ISSUE_TEMPLATE.md]: If you are having paid support => If you have paid support

File: .github/ISSUE_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+     business please go to https://scc.suse.com/support for support.
```
Comment (lagartoflojo) [inline .github/ISSUE_TEMPLATE.md]: I'd rephrase this as:

If you have paid support from SUSE, go to https://scc.suse.com/support to open a support ticket.

File: .github/ISSUE_TEMPLATE.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+   - When posting bugs/feedback here, there is no guarantee that the issue
+     will be handled immediately. Depending on the topic and the availability
+     of the developers, the issue maybe resolved in undefined time.
```
Comment (lagartoflojo) [inline .github/ISSUE_TEMPLATE.md]: I'd get rid of the second sentence (Depending on the topic... undefined time.) It basically repeats the first sentence.

File: .github/ISSUE_TEMPLATE.md (hunk 7 above)
Comment (lagartoflojo) [reply .github/ISSUE_TEMPLATE.md]: > publicly accessible
